inference-engine/src/vpu/graph_transformer/src/frontend/in_out_convert.cpp

```diff
@@ -65,7 +65,8 @@ void FrontEnd::addDataTypeConvertStages(const Model& model) {
             }
 
             case DataType::U8:
-            case DataType::FP32: {
+            case DataType::FP32:
+            case DataType::S32: {
```

Comment:
Why you do this? `S32` input is supposed to be processed as is.
@taka-no-me please share your opinion there

Reply:
Reverted this change. Let's discuss it separately.